run.py

```diff
@@ -237,6 +237,7 @@ def create_nodes(
                     private_ip=private_ip,
```

Comment:
this change should go here - https://github.com/red-hat-storage/cephci/pull/3157